packages/grafana-api-clients/src/clients/rtkq/notifications.alerting/v0alpha1/endpoints.gen.ts

```diff
@@ -119,44 +119,6 @@ const injectedRtkApi = api
         }),
         invalidatesTags: ['Receiver'],
       }),
-      getReceiverStatus: build.query<GetReceiverStatusApiResponse, GetReceiverStatusApiArg>({
```

Comment:
Are the (deletion) diffs here just because the spec had drifted from the latests generated code?

Reply:
I'll chalk this up to changes in the app-sdk tool and we've probably never re-generated the client.